inexor_unix

```diff
@@ -1,23 +1,9 @@
 #!/bin/bash
-tolower() {
-  tr '[:upper:]' '[:lower:]'
-}
-
-arch="`uname -m | tolower`"
-# Rename the arch to bare style
-if [ "${arch}" = "x86_64" ]
-then
-  arch="64"
-else
-  arch="32"
-fi
 
 dir="`dirname "$0"`"
-os="`uname | tolower`"
```

Comment:
With this PR everything will go into the direction of "do just one thing at a time".
Therefore on Windows you can't switch between the different configurations (Debug/Release) inside the VS-IDE anymore, simply because conan.io doesn't (yet probably) support it.

The folder structure inside bin was an unnecessary complexity which made e.g. specifying the path for CEF overly hard, while providing no benefit. People don't want "universal" packages, they want one for their specific distro.
(+ Installation would have been harder with conan. we would have needed to split things up manually again in bin/all bin/<os>/<arch )